### ISSUE TITLE
State of this repository

### ISSUE THREAD

**Issue by thomasdondorf** [thomasdondorf/puppeteer-cluster#428]
### TLDR: I will probably be able to keep the dependencies up to date and might also merge good looking pull requests. Just don't expect a lot of new features.

---

When I published this repository I never expected that it would be used by that many people. That said, I haven't used puppeteer for a long time. I'm willing to put some work into this, but I'm doing this completely in my free time. After more than one year, today I've release a new version with up-to-date dependencies.

**Future maintenance of this repository**  
I don't have a lot of interest in adding features anymore, but I will probably be able to update the dependencies from time to time. I might also check issues and pull requests. If anyone wants to help out, I would be happy to add them as contributor. The testing/release process is basically automated.

**I'm not a support hotline**
I get a few mails every month from people who expect free support from me. I stopped answering these mails a while ago. If you need my help in your project or you have trouble to get the library to work at your company, make it clear in your mail that you are willing to pay. Otherwise, I will ignore your mail. Sorry if that sounds harsh to you, but publishing this library does not mean I owe you support.

**Is it safe to keep using `puppeteer-cluster` and what are alternatives?**
The only alternative I'm aware of is [Apify](https://github.com/apify/apify-js). But it's more an "all-in-one" solution for different crawling scenarios and not as focused on the pooling/clustering as this repo. So if `puppeteer-cluster` fits your use case, you can keep using it. I don't see why you shouldn't. Just don't expect new features.

That said, thanks for using this library. ❤️

**Comment by seyfer** [comment]
@thomasdondorf please make it visible in the main README file, to make it clear. 
Some developers lose time trying to adopt it, not knowing that development stopped here.

**Comment by jacob-hyde** [comment]
@thomasdondorf I would like to be a maintainer of this project. I am using it for a few scrapers. Anyone else want to help?
